packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/email-action/components/WorkflowSendEmailBody.tsx

```diff
@@ -0,0 +1,265 @@
+      readonly,
+      defaultValue,
+      onUpdate: (editor) => {
+        const jsonContent = editor.getJSON();
+        onChange(JSON.stringify(jsonContent));
+      },
+      onFocus: () => {
+        pushFocusItemToFocusStack({
+          focusId: instanceId,
+          component: {
+            type: FocusComponentType.FORM_FIELD_INPUT,
+            instanceId: instanceId,
+          },
+          globalHotkeysConfig: {
+            enableGlobalHotkeysConflictingWithKeyboard: false,
+          },
+        });
+      },
+      onBlur: () => {
+        removeFocusItemFromFocusStackById({ focusId: instanceId });
+      },
+      onImageUpload: handleUploadAttachment,
+    },
+    [isFullScreen],
+  );
+
+  const handleEnterFullScreen = () => {
+    setIsFullScreen(true);
+  };
+
+  const handleExitFullScreen = () => {
+    setIsFullScreen(false);
+  };
+
+  const handleVariableTagInsert = (variableName: string) => {
+    if (!isDefined(editor)) {
+      throw new Error(
+        'Expected the editor to be defined when a variable is selected',
+      );
+    }
+
+    editor.commands.insertVariableTag(variableName);
+  };
+
+  const breadcrumbLinks: BreadcrumbProps['links'] = [
+    {
+      children: workflow?.name?.trim() || t`Untitled Workflow`,
+      href: '#',
+    },
+    {
+      children: headerTitle,
+      href: '#',
+    },
+    {
+      children: t`Email Editor`,
+    },
+  ];
+
+  const fullScreenOverlay = isFullScreen
+    ? createPortal(
+        <StyledFullScreenOverlay
+          data-globally-prevent-click-outside="true"
+          tabIndex={-1}
+        >
+          <StyledFullScreenHeader
+            title={<Breadcrumb links={breadcrumbLinks} />}
+            hasClosePageButton={!isMobile}
+            onClosePage={handleExitFullScreen}
+          />
+          <StyledFullScreenContent data-globally-prevent-click-outside="true">
+            <StyledFullScreenEmailEditorContainer>
+              <WorkflowEmailEditor editor={editor} readonly={readonly} />
+            </StyledFullScreenEmailEditorContainer>
+          </StyledFullScreenContent>
+        </StyledFullScreenOverlay>,
+        document.body,
+      )
+    : null;
+
+  if (!isDefined(editor)) {
+    return null;
+  }
+
+  return (
+    <>
+      <StyledWorkflowSendEmailBodyContainer>
+        {label ? <InputLabel>{label}</InputLabel> : null}
+
+        <StyledWorkflowSendEmailFieldContainer>
+          <StyledWorkflowSendEmailBodyInnerContainer>
+            {!isFullScreen && (
+              <WorkflowEmailEditor editor={editor} readonly={readonly} />
+            )}
+
+            <StyledEmailEditorActionButtonContainer>
+              {!readonly && !isFullScreen && (
+                <StyledFullScreenButtonContainer
+                  isUnfolded={false}
+                  transparentBackground
+                  onClick={handleEnterFullScreen}
+                >
+                  <IconMaximize size={theme.icon.size.sm} />
```

Comment:
There is an anesthetic alignment issue, probably it is because this should be md and not sm